tests/foreman/cli/test_repository.py

```diff
@@ -560,6 +562,41 @@ def test_positive_create_docker_repo_with_name(self, repo_options, repo):
         for key in 'url', 'content-type', 'name':
             assert repo.get(key) == repo_options[key]
 
+    @pytest.mark.tier1
+    def test_positive_create_with_new_organization_and_location(self, default_sat):
+        """Check if error is thrown when creating a Repo with a new Organization and Location.
+
+        :id: 9ea4f2a9-f339-4215-b301-cd39c6b5c474
+
+        :parametrized: no
+
+        :expectedresults: No error is present when Repository is created
+
+        :BZ: 1992967
+
+        :CaseImportance: Critical
```

Comment:
```suggestion
        :CaseImportance: High
```

because bug comment 0 says " I can see that the repository is created despite of the error shown."

Reply:
and because there are already lots of Critical tests that create repos